lib/Sema/CSSimplify.cpp

```diff
@@ -2077,6 +2077,42 @@ static ConstraintFix *fixRequirementFailure(ConstraintSystem &cs, Type type1,
   llvm_unreachable("covered switch");
 }
 
+static ConstraintFix *fixPropertyWrapperFailure(
+    ConstraintSystem &cs, Type baseTy, Expr *anchor, ConstraintLocator *locator,
+    llvm::function_ref<bool(ResolvedOverloadSetListItem *, VarDecl *, Type)>
+        attemptFix,
+    Optional<Type> toType = None) {
+  if (auto resolvedOverload = cs.findSelectedOverloadFor(anchor)) {
```

Comment:
Nit: It'd be great to do an early return here if `resolvedOverload` couldn't be identified. That way you identify exit conditions early and don't have to indent the rest of the function, so it makes code much easier to read.